packages/e2e-tests/cypress/support/householder-planning/appeals-service/page-objects/householder-planning-listed-building-po.js

```diff
@@ -0,0 +1,3 @@
+export const getIsListedBuilding = () => cy.get('input[data-cy="answer-full-planning"]');
```

Comment:
This needs to move under packages\e2e-tests\cypress\support\eligibility\page-objects

Reply:
Moved and renamed.